tf-static-website/README.md

```diff
@@ -0,0 +1,30 @@
+# Setup Serverless Blog Architecture
+
+These terraform scripts will set up a new serverless blog architecture. There are several manual steps below. In this case, *terraform-init-user* is used as an example of an IAM user.
+
+## Getting Started
+### Apply the terraform scripts
+1. Run `$ AWS_PROFILE=terraform-init-user terraform init -backend-config=backend-config/s3.tfbackend`
```

Comment:
I think you're missing an instruction here, because you don't mention how to modify the s3.tfbackend file.

Reply:
I'll add this.